packages/build-utils/src/fs/run-user-scripts.ts

```diff
@@ -333,11 +334,8 @@ export async function scanParentDirs(
     cliType = 'npm';
     lockfilePath = npmLockPath;
     lockfileVersion = packageLockJson.lockfileVersion;
-  } else if (bunLockBin) {
-    cliType = 'bun';
-    lockfilePath = bunLockPath;
-    // TODO: read "bun-lockfile-format-v0"
-    lockfileVersion = 0;
+  } else {
+    cliType = 'npm';
```

Comment:
<!-- Questions comments seek clarification.  -->
**question (blocking):** ‏Can we wrap this in a flag we can flip?

Reply:
Yep good point! I added a flag `VERCEL_ENABLE_NPM_DEFAULT` in the latest commit